todo_app/app.py

```diff
@@ -1,4 +1,8 @@
-from flask import Flask
+from flask import Flask, render_template,request
+from flask.ctx import RequestContext
+from flask.wrappers import Request
```

Comment:
Unused imports.

In general if you see something greyed out in VS Code, that means it's not used and can be deleted.